### ISSUE TITLE
Voice call to multiple phone numbers at a single API call

### ISSUE THREAD

**Issue by tobslob** [twilio/voice-quickstart-server-node#21]
Do Twilio voice call API support multiple voice calls at the same time or map concurrently through the API to send out multiple calls?

**Comment by bobiechen-twilio** [comment]
Hi @tobslob 

Thanks for reaching out.
Trying to understand your question correctly - are you trying to set up and have multiple concurrent calls at the same time? It would be great if you could share more detail about your use case (also the platform, e.g. mobile or JS, rtc.

**Comment by tobslob** [comment]
Yes, I want to able to call 10, 000 phone numbers at the same time or concurrently, and they should able to pick up the calls and listen to the TwiML.

**Comment by bobiechen-twilio** [comment]
Thanks @tobslob. Both `Conference` and `Call` APIs (or TwiML verbs) allow reaching to multiple clients at the same time.
For Conference, check out the [Participant Resource API](https://www.twilio.com/docs/voice/api/conference-participant-resource).
As for the Call API, Twilio Programmable Voice allows applications to setup multiple client device (JS) or register mobile client push notification bindings with the same identity. All devices will ringing when a Call is made to this client identity, but only one of them can answer/reject the call.

For detailed help to set up your application logic, please contact [Twilio Support](https://support.twilio.com/hc/en-us).

**Comment by tobslob** [comment]
What's the best option to explore if I want to able to send out multiple calls up to 10,000 and I want them to listen to the same TwiML verbs?

**Comment by bobiechen-twilio** [comment]
The best way for all clients be able to join (listen) to the same TwiML would be to create a <Conference> room and send the conference SID to the clients via whichever communication channels that work for your application logic, e.g. regular push notification or websocket.

**Comment by hamayun53** [comment]
@bobiechen-twilio 
Hi buddy! 
I have a use case, I am working on a mobile application and I want to register more than one number for a single device in such a way to make user able to recieve call and message on any of te Twillio registered number which is associated with our mobile application on the device.

How to make this possible, kindly guide me.
